src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.html

```diff
@@ -0,0 +1,117 @@
+<div class="col-sm-12 col-lg-6">
+  <form class="form-horizontal"
+        #frm="ngForm"
+        [formGroup]="userForm"
+        novalidate>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 i18n="form title|Example: Create Pool@@formTitle"
+            class="panel-title">{{ action | titlecase }} {{ resource | upperFirst }}</h3>
+      </div>
+      <div class="panel-body">
+
+        <!-- Old password -->
+        <div class="form-group"
```

Comment:
To have a horizontal form we now need to add the `row` class to each `form-group`.
```suggestion
        <div class="form-group row"
```